### PR TITLE
fix: positions table close/action column title

### PR DESCRIPTION
**Pull request by aforaleka**
pre isolated margin:
image
image
image
post isolated margin:
image
image
image

### CODE REVIEW

File: src/views/tables/PositionsTable.tsx [hunk 1]
```diff
@@ -335,7 +335,9 @@ const getPositionsTableColumnDef = ({
           key:
             shouldRenderTriggers && showClosePositionAction && !testFlags.isolatedMargin
               ? STRING_KEYS.ACTIONS
-              : STRING_KEYS.CLOSE,
+              : showClosePositionAction
+              ? STRING_KEYS.CLOSE
+              : STRING_KEYS.ACTION,
```
Comment (yogurtandjam) [inline src/views/tables/PositionsTable.tsx:340]: [nit] can we define this variable somewhere else/abstract this to a function? this ternary is getting pretty long and a little difficult to read

File: src/state/accountCalculators.ts [hunk 2]
```diff
@@ -103,5 +105,7 @@ export const calculateShouldRenderActionsInPositionsTable = (isCloseActionShown:
   createSelector(
     [calculateIsAccountViewOnly, calculateShouldRenderTriggersInPositionsTable],
     (isAccountViewOnly: boolean, areTriggersRendered: boolean) =>
-      !isAccountViewOnly && (areTriggersRendered || isCloseActionShown)
+      !isAccountViewOnly &&
+      ((!testFlags.isolatedMargin && (areTriggersRendered || isCloseActionShown)) ||
+        (testFlags.isolatedMargin && isCloseActionShown))
```
Comment (yogurtandjam) [inline src/state/accountCalculators.ts:110]: [nit] i feel like if we un-inline this it will be easier to understand what cases have what results

**Review by yogurtandjam — APPROVED** [review]
AWN

**Review by jaredvu — APPROVED** [review]

File: src/views/tables/PositionsTable.tsx (hunk 1 above)
Comment (aforaleka) [reply src/views/tables/PositionsTable.tsx:340]: leaving this as is for now -- will follow up when we clean up the `shouldRenderTriggers` variable which should help

**Pull request merged**